core/src/main/java/org/polypheny/db/util/background/BackgroundTask.java

```diff
@@ -15,22 +15,31 @@
 
 
     enum TaskSchedulingType {
-        EVERY_SECOND( 1000 ),
-        EVERY_FIVE_SECONDS( 5000 ),
-        EVERY_TEN_SECONDS( 10000 ),
-        EVERY_THIRTY_SECONDS( 30000 ),
-        EVERY_MINUTE( 60000 ),
-        EVERY_TEN_MINUTES( 600000 ),
-        EVERY_FIFTEEN_MINUTES( 900000 ),
-        EVERY_THIRTY_MINUTES( 1800000 );
+        EVERY_SECOND( 1000, TaskDelayType.DELAYED ),
```

Comment:
Adjust

Reply:
Now the enum includes:
       EVERY_SECOND( 1000, TaskDelayType.DELAYED ) -> for all time possibilities with TaskDelayType.DELAYED

       EVERY_SECOND_FIXED( 1000, TaskDelayType.FIXED ) -> for all time options with TaskDelayType.FIXED